Source/Core/Common/DebugInterface.h

```diff
@@ -38,6 +38,7 @@ class DebugInterface
   // Memory Patches
   virtual void SetPatch(u32 address, u32 value) = 0;
   virtual void SetPatch(u32 address, std::vector<u8> value) = 0;
+  virtual const Common::Debug::MemoryPatch& GetPatch(std::size_t index) const = 0;
```

Comment:
MemoryPatch

Reply:
Unless I'm mistaken, I can't do that. The `DebugInterface` class isn't part of the `Common` namespace.